packages/terra-worklist-data-grid/tests/wdio/worklist-data-grid-spec.js

```diff
@@ -110,7 +131,181 @@ Terra.describeViewports('WorklistDataGrid', ['medium', 'large'], () => {
         .concat(['Control', 'End', 'Control'])
         .concat(new Array(3).fill('ArrowRight')));
 
-      Terra.validates.element('last-cell-in-last-row-selected', { selector });
+      Terra.validates.element('last-cell-in-last-row-focused', { selector });
+      expect(browser.$$('[role="grid"] [tabIndex="0"]')).toBeElementsArrayOfSize(1);
+    });
+  });
+
+  describe('cell selection : ', () => {
+    beforeEach(() => {
+      browser.url('/raw/tests/cerner-terra-framework-docs/worklist-data-grid/default-worklist-data-grid');
+    });
+
+    it('spacebar selects a cell', () => {
```

Comment:
Test descriptions should read as a sentence starting with it. e.g.
"it selects a cell with spacebar"

```suggestion
    it('selects a cell with spacebar', () => {
```

Reply:
Thanks for pointing this out. I changed the others to read accordingly.